src/parser/parser.cc

```diff
@@ -524,12 +524,12 @@ class Parser {
   NDArray NumberToNDArray(const Token& token) {
     if (token->token_type == TokenType::kInteger) {
       DLDevice dev = {DLDeviceType::kDLCPU, 0};
-      auto dtype = String2DLDataType("int32");
+      auto dtype = String2DLDataType("int64");
       auto data = NDArray::Empty({}, dtype, dev);
-      auto array = reinterpret_cast<int32_t*>(data->data);
+      auto array = reinterpret_cast<int64_t*>(data->data);
       // revisit this, literal node issue.
       int64_t value = Downcast<tvm::Integer>(token->data);
-      array[0] = (int32_t)value;
+      array[0] = (int64_t)value;
```

Comment:
`static_cast<int64_t>`(always prefer c++ style cast)

Reply:
it looks like we don't even need this particular cast now